lightcone/plot_3d.py

```diff
@@ -317,3 +328,44 @@ def match_2d_plot_to_3d_plot(self, fig, plot_2d, alpha=1):
         ax.invert_xaxis()
 
         return fig
+
+    def color_lens_and_source(self, config_handler, sample, noised_image=True, artifact_image=False):
```

Comment:
could you structure the code such that one does not have to rely on a paltas config_handler when using the color scheme? Would be nice if this could be used also for the more simple notebooks without using paltas.